tools/dts.js

```diff
@@ -8,7 +8,7 @@ var singleLineComment = /\/\/[^\n]*\n/g;
 var multiLineComment = /(^\/\*(\*(?!\/)|[^*])*\*\/\s*)/m;
 
 // 1. [export ][default |declare ](class|interface) <NAME>[ extends| implements <BASECLASS>] {...}
```

Comment:
Curious, why did the regex need to be changed?

Reply:
in order to match multiple extending, such as
interface MyInterface extends Interface1, Interface2, Interface3 { ...